### DISCUSSION TITLE
为什么按照教程走完天气还是没有效果

### DISCUSSION THREAD

**Discussion by AixLeftY** [NSRingo/.github#16] [Q&A 问答] [answered]
用圈x导入规则，然后也加了js订阅设置好了，也删除过天气app重启过都没有生效，看了重写规则有weatherkit这条规则日志

**Comment by VirgilClyne** [comment, marked as answer]
移除你QX配置文件中`[mitm]`段落`hostname`中的 `-*.apple.com`

**Reply by AixLeftY** [reply]
麻烦再问一下，移除这个会有什么影响吗

**Reply by VirgilClyne** [reply]
谁在你配置里加的这条，你去问他

**Reply by Jackie512** [reply]
感谢🙏！！我也遇到了这个问题，解决了已经
